platform/backend/repository/testbed/src/test/kotlin/io/hamal/repository/CodeTest.kt

```diff
@@ -418,6 +417,41 @@ internal class CodeRepositoryTest : AbstractUnitTest() {
 
     }
 
+
+    @Nested
+    inner class CodeTypeTest {
+        @TestFactory
+        fun `Set code type`() = runWith(CodeRepository::class) {
+            createCode(
+                codeId = CodeId(1),
+                groupId = GroupId(3),
+                codeValue = CodeValue("1 + 1")
+            )
+            val c = find(CodeId(1))!!
+            c.type = CodeType.None
+            assertThat(c.type, equalTo(CodeType.None))
+        }
+
+        @TestFactory
+        fun `Get code type by id`() = runWith(CodeRepository::class) {
+            createCode(
+                codeId = CodeId(1),
+                groupId = GroupId(3),
+                codeValue = CodeValue("1 + 1")
+            )
+            val c = find(CodeId(1))!!
+            assertThat(c.type, equalTo(CodeType.of(c.type.value)))
+        }
```

Comment:
```suggestion
```

I dont see any value in this tests - what did you try to accomplish here?

Essentially what you are testing here is

```
val code =  Code(...)
code.type = CodeType.None
code.type, equalTo(CodeType.None))
```

We dont need to test getters and setters